science/exe.py

```diff
@@ -159,16 +175,7 @@ def _export(
 
         lift_manifest = chroot / "lift.json"
 
-        build_info = dict[str, Any]()
-        if include_provenance:
-            build_info.update(
-                note="Generated by science.",
-                version=__version__,
-                url=(
-                    f"https://github.com/a-scie/lift/releases/tag/v{__version__}/"
-                    f"{Platform.current().qualified_binary_name('science')}"
-                ),
-            )
+        build_info = gather_build_info(lift_digest) if include_provenance else None
```

Comment:
For science itself, looks like so:

## Add provenance to build (the release will do this)
```
$ dist/science build --include-provenance --file science.pyz=dist/science.pyz --dest-dir dist/provenance
dist/provenance/science
```

## The resulting extra scie-jump lift manifest data
```json
$ SCIE=inspect dist/provenance/science | jq .science
{
  "binary": {
    "hash": "c15949fd61685be0b109363301c9740e70b43536f581758c34fc7ba83cc93bfc",
    "size": 6463256,
    "url": "https://github.com/a-scie/lift/releases/tag/v0.1.0/science-linux-x86_64",
    "version": "0.1.0"
  },
  "git_state": "ee2ee6e",
  "note": "This scie lift JSON manifest was generated from a source lift toml manifest using the science binary.",
  "source_lift_toml": {
    "hash": "49dc36a6db71bccf1bff35363454f7567fd124ba80d1e488bd320668a11c70bc",
    "size": 432
  }
}
```